src/main/java/org/reaktivity/nukleus/oauth/internal/stream/OAuthProxyFactory.java

```diff
@@ -333,6 +342,31 @@ private RouteFW wrapRoute(
         return routeRO.wrap(buffer, index, index + length);
     }
 
+    private long resolveChallengeResponseTimeout(
+        JsonWebSignature verified)
+    {
+        long challengeResponseTimeout = 0;
+
+        try
+        {
+            if (verified != null)
+            {
+                final JwtClaims claims = JwtClaims.parse(verified.getPayload());
+                final NumericDate challengeAfterDate = claims.getNumericDateClaimValue(
```

Comment:
Suggest renaming this temporary stack variable to `claimValue` since it is no longer related to `challengeAfter` which was absolute.